File: cash.c
#include <cs50.h>
#include <stdio.h>
#include <math.h>

int main(void)
{
    float cash;
    do 
    { 
        cash = get_float("Cash: ");
    }
    while (cash <= 0.009);
    
    int cents = round(cash * 100);
 
    int c25 = cents / 25; 
    int c25r = cents % 25;
   
    int c10 = c25r / 10; 
    int c10r = c25r % 10;
    
    int c5 = c10r / 5; 
    int c5r = c10r % 5;
    
    int c1 = c5r / 1;
    
    printf("%i\n", (c25 + c10 + c5 + c1));
    
}

File: pset6/Cash/cash.py
from cs50 import get_float

# request input from user
while True:
    n = get_float('Change owed: ')
    if n > 0:
        break

c = round(n * 100)
# check 25c
n25 = c // 25
c25 = c % 25
# check 10c
n10 = c25 // 10
c10 = c25 % 10
# check 5c
n5 = c10 // 5
c5 = c10 % 5
# check 1c
n1 = c5

numb = n25 + n10 + n5 + n1
print(numb)